### PR TITLE
[Hold] Platform Outlook Source Connector docs

### PR DESCRIPTION
**Pull request by ds-filipknefel**
Documentation for Platform Outlook Source Connector

Do not publish until the connector is visible and usable in production.

Internal staged build of the doc: https://unstructured-53-platform-outlook-source-connector.mintlify.app/platform/sources/outlook

### CODE REVIEW

File: snippets/general-shared-text/outlook-platform.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Fill in the following fields:
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook-platform.mdx:1]: @ds-filipknefel Please confirm that my edits are technically correct.

File: snippets/general-shared-text/outlook.mdx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The Outlook prerequisites:
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook.mdx:1]: @ds-filipknefel Please fill in this list of user prerequisites in order to get what they need to fill in the fields in the UI. For each prerequisite, please link to public third-party step-by-step documentation that instructs the user how to get these details.

Comment (ds-filipknefel) [reply snippets/general-shared-text/outlook.mdx:1]: I've included an Azure registration link, a link to official Microsoft guide on how to create the Application (also a video, though it doesn't cover the guide exactly which I mention). I've also added info on where values of few fields related to the application in question can be found.

File: platform/sources/outlook.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<OutlookPrerequisites />
+
+To create the source connector:
+
+1. On the sidebar, click **Sources**.
+2. Click **New Source**.
+3. In the **Type** drop-down list, select **Outlook**.
+4. Fill in the fields as described later on this page.
+5. Click **Save and Test**.
+6. Click **Close**.
```
Comment (Paul-Cornell) [inline platform/sources/outlook.mdx:20]:
```suggestion
1. On the sidebar, click **Connectors**.
2. Click **Sources**.
3. Click **Add new**.
4. Give the connector some unique **Name**.
5. In the **Provider** area, click **Outlook**.
6. Click **Continue**. 
7. Follow the on-screen instructions to fill in the fields as described later on this page.
8. Click **Save and Test**.
```

File: snippets/general-shared-text/outlook-platform.mdx [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+- **Name** (_required_): A unique name for this connector.
+- **User Email** (_required_): The Outlook email account address to access messages from.
+- **Folders** (_required_): A comma-separated list of email folders to download messages from. To access messages from subfolders as well, check the **Recursive** box.
+- **Client ID** (_required_): The client ID of the Microsoft Entra ID application that has access to the Outlook email account address. Can be found in App registrations' **Overview** sections of your application.
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook-platform.mdx:6]:
```suggestion
- **Client ID** (_required_): The client ID of the Microsoft Entra ID application that has access to the Outlook email account address. This client ID can be found in the app registration's **Overview** section in the Azure portal.
```

File: snippets/general-shared-text/outlook-platform.mdx [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+- **Client Credential** (_required_): The secret ID of the corresponding Entra ID application. Can be read from App registrations' **Cerificates & secrets** sections of your application **only** right after creation.
+- **Authentication Provider URL**: The authentication token provider for the Entra ID application. If this field is left blank, `https://login.microsoftonline.com` is used by default.
+- **Tenant**: The ID or domain name that is associated with the Entra ID application. If this field is left blank, `common` is used by default. Can be found in App registrations' **Overview** sections of your application.
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook-platform.mdx:9]:
```suggestion
- **Tenant**: The ID or domain name that is associated with the Entra ID application. If this field is left blank, `common` is used by default. This value can be found in the app registration's **Overview** section in the Azure portal.
```

File: snippets/general-shared-text/outlook.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+
+A Microsoft Entra ID tenant. If you don't have a tenant, [create an Azure account](https://azure.microsoft.com/en-us/pricing/purchase-options/azure-account/).
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook.mdx:4]:
```suggestion
A Microsoft Entra ID tenant. If you don't have a tenant, [create an Azure account](https://azure.microsoft.com/pricing/purchase-options/azure-account/).
```

File: snippets/general-shared-text/outlook.mdx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+width="560"
+height="315"
+src="https://www.youtube.com/embed/2bQ6WiJ1ncA"
+title="YouTube video player"
+frameborder="0"
+allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+allowfullscreen
+></iframe>
+
+An Azure identity platform managed application. Learn how to create one from **Register an application** and **Add credentials (Option 2)** steps from Microsoft's [official guide](https://learn.microsoft.com/en-us/graph/auth-register-app-v2). Watch how to **Register an application** in the following video, you can omit the **Redirect URI** part and still need to add credentials:
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook.mdx:16]:
```suggestion
A Microsoft Entra ID app registration. [Create an app registration](https://learn.microsoft.com/graph/auth-register-app-v2). See also the following video, but note that while you can omit the **Redirect URI** part, you must still add credentials:
```

File: snippets/general-shared-text/outlook.mdx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+src="https://www.youtube.com/embed/au1dBzirmvQ?si=hRakOvOI4oxZ56qv" 
+title="YouTube video player"
+frameborder="0"
+allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin"
+allowfullscreen>
+</iframe>
+
+Once you've created the application you can access data for **Client ID** and **Tenant** fields in App registrations' **Overview** section.
+
+Copy and store the secret created in App registrations' **Certificates & secrets > Client secrets > New client secret.** for the **Client Credential** field.
```
Comment (Paul-Cornell) [inline snippets/general-shared-text/outlook.mdx:30]:
```suggestion
Once you've created the app registration, you can get the **Client ID** and **Tenant** values from the app registration's **Overview** section in the Azure portal.

Copy and store the secret that you create by using the app registration's **Certificates & secrets > Client secrets > New client secret** link. The secret will appear in the **Client Credential** field.
```

**Comment by Paul-Cornell** [comment]
Duplicated by https://github.com/Unstructured-IO/docs/pull/343

**Pull request closed without merging**